module/lua/lstrlib.c

```diff
@@ -853,9 +853,9 @@ static void addquoted (lua_State *L, luaL_Buffer *b, int arg) {
     else if (*s == '\0' || iscntrl(uchar(*s))) {
       char buff[10];
       if (!isdigit(uchar(*(s+1))))
-        sprintf(buff, "\\%d", (int)uchar(*s));
+		  snprintf(buff, sizeof(buff), "\\%d", (int)uchar(*s));
```

Comment:
nit: preserve the existing indentation of the original lua code.

Reply:
Oh hmm its all spaces.